pkg/ccl/changefeedccl/encoder_test.go

```diff
@@ -831,30 +786,25 @@ func TestAvroMigrateToUnsupportedColumn(t *testing.T) {
 		}
 	}
 
-	t.Run(`sinkless`, sinklessTest(testFn))
-	t.Run(`enterprise`, enterpriseTest(testFn))
+	t.Run(`kafka`, kafkaTest(testFn))
 }
 
 func TestAvroLedger(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
 	testFn := func(t *testing.T, db *gosql.DB, f cdctest.TestFeedFactory) {
-		reg := cdctest.StartTestSchemaRegistry()
```

Comment:
also don't need a separate schema registry since kafkafeed creates one automatically which will override this one. if we need to check the sink's registry we can directly access it as a member of kafkafeed.